source/release-notes/v1.7-release-notes.rst

```diff
@@ -289,4 +291,18 @@ Administrators can now disable the link that appears in the batch connects' card
 node that the job is running on. This is helpful for sites that don't allow regular users to shell into
 compute nodes.
 
-To do so, simply set ``OOD_BC_SSH_TO_COMPUTE_NODE=0`` (or 'false' or 'off') in ``/etc/ood/config/apps/dashboard/env``.
\ No newline at end of file
+To do so, simply set ``OOD_BC_SSH_TO_COMPUTE_NODE=0`` (or 'false' or 'off') in ``/etc/ood/config/apps/dashboard/env``.
+
+Reduced rpm size by 40 percent
+..............................
+
+We reduced the size of the main OnDemand rpm by nearly 40%.
+
+The 1.6 ondemand-1.6.22-1.el7.x86_64.rpm was 162M and in 1.7 this has been reduced by 63M to a total of 99M, split between two rpms: ondemand-gems-1.7.10-1.7.10-2.el7.x86_64.rpm with 68M, and ondemand-1.7.10-2.el7.x86_64.rpm with 31M. This is in part due to switching to a monorepo at https://github.com/osc/ondemand and installing all the gems into a central shared location, installed using the new ondemand-gems rpm.
+
+New shared OnDemand gem sets
+............................
+
+The new ondemand-gems rpms allow other Passenger apps to be developed using the same gems that the core OnDemand apps use. If you do ``scl enable ondemand`` and then ``gem list`` you can see all the gems available.
+
+Because the ondemand-gems rpms will include the version in the gem name, it is possible to install multiple versions of these rpms on the same system. For example, "ondemand-gems-1.7.10" is the name and "1.7.10-2.el7.x86_64.rpm" the version of ondemand-gems-1.7.10-1.7.10-2.el7.x86_64.rpm. That way you can upgrade to newer versions of OnDemand but still have the ability to provide apps access to older gems by installing the older gem sets.
```

Comment:
Replace `include the version in the gem name` with `include the version in the RPM name`.

Also `rpms` should be `RPMs`.